run.sh

```diff
@@ -65,14 +65,14 @@ done
 GALAXY_VIRTUAL_ENV="${GALAXY_VIRTUAL_ENV:-.venv}"
 if [ -d "$GALAXY_VIRTUAL_ENV" -a -z "$skip_venv" ];
 then
-    [ -n "$PYTHONPATH" ] && { echo 'Unsetting $PYTHONPATH'; unset PYTHONPATH; }
```

Comment:
I prefer the old syntax here (and below on line 75) as the single quotes are used explicitly because we do *not* want variable expansion inside this string. Switching to double quotes and adding an escape seems more cumbersome.